ynput/OpenPype
# Maya: Remove RenderSetup layer observers

## Changelog Description

Remove RenderSetup layer observers that are not needed since new publisher since Renderlayer Creators manage these themselves on Collect and Save/Update of instances.

## Additional info

🔥 
🗑️ 

## Testing notes:

1. Create, manage and publish renderlayers
2. Make sure to also open a few different scenes
3. Make sure to also create a few layers, remove layers, etc.
